fix(world): prevent initialising the world multiple times [L-05] (#2170)
latticexyz/mud: .changeset/neat-tools-check.md

---
"@latticexyz/world": patch
---

> Added a table that tracks whether the world has already been initialized.

Reviewer: ```suggestion
Added a table to track the `CoreModule` address the world was initialised with.
```
